### ISSUE TITLE
The ability to replace BrowserRouter with just Router

### ISSUE THREAD

**Issue by ralexandr** [reactql/kit#68]
Hi, thank you for your great kit and for making our lifes a little bit simplier 👍

In my project I need to access browserHistory object to manage navigation inside my redux actions.
I've replaced BrowserRouter with Router inside 'browser' entry and passed history object created with 'history/createBrowserHistory' to Router component as a prop.
And now I'm able to control this history my importing the same history object inside my redux actions.

But the problem is that I've changed reactql's file 'kit/entry/browser.js' and on next update my changes will be overwritten.

Could you please give the out-the-box ability to get browser history and import it anywhere in my react app?

**Comment by leebenson** [comment]
I'll explore a better pattern for this in the next release

**Comment by klarstrup** [comment]
Is this different from `history` provided by `react-router`'s `withRouter` HoC?

**Comment by ralexandr** [comment]
> Is this different from history provided by react-router's withRouter HoC?

No, but i can't use `withRouter` in my redux actions directly. I'm forced to wrap with `withRouter` my react component and pass `history` object from component's props to redux action. 
I want to keep my component simple, don't use `withRouter` and get `history` directly inside my redux action.

**Comment by klarstrup** [comment]
Ah, I missed the redux action part! :thinking: Carry on

**Comment by arsanjea** [comment]
I second ralexandr's request. This is a key missing feature that takes a few seconds to add if I'm not missing anything:

In kit/lib/routing.js:


```
import createHistory from 'history/createBrowserHistory';
import { Route, Redirect as ReactRouterRedirect, Router } from 'react-router-dom';

export const browserHistory = createHistory();
...

export const RouterWithHistory = props => (
  <Router history={browserHistory}>
    {props.children}
  </Router>
);
```


in kit/entry/browser.js:

```
import { RouterWithHistory } from '../lib/routing';

...

  const Chain = () => (
    <ApolloProvider store={store} client={client}>
      <RouterWithHistory>
        <App />
      </RouterWithHistory>
    </ApolloProvider>
  );

```

in src/components/main/index.js (test):
```
// NotFound 404 handler for unknown routes
import { Redirect, browserHistory } from 'kit/lib/routing';
...
    <ul>
      <li><Link to="/">Home</Link></li>
      ....
      <li><button onClick={() => browserHistory.push('/page/about')}>Test About</button></li>
    </ul>

```

**Comment by leebenson** [comment]
Released in [2.9.3](https://github.com/reactql/kit/releases/tag/2.9.3).

[kit/lib/routing.js](https://github.com/reactql/kit/blob/master/kit/lib/routing.js) now exports a `history` which is injected into the browser entry point.

I've included an example in the out-the-box project on fresh installs, which shows `history.push()` being invoked directly.
